packages/vscode-extension/package.json

```diff
@@ -102,6 +200,8 @@
   },
   "dependencies": {
     "@neo4j-cypher/language-server": "2.0.0-next.7",
+    "@neo4j-cypher/schema-poller": "2.0.0-next.6",
+    "neo4j-driver": "^5.12.0",
```

Comment:
I wonder if it might be better to create a "persistent connection" object that wraps the driver so we don't need to expose the vscode extension to this level of abstraction